### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by fire717** [YZY-stack/Effort-AIGI-Detection#9]
hi!

In the effort_detector.py,  line142
```
# loss2 = self.compute_weight_loss()
 # overall_loss = loss + loss2
```
loss2 is not use, so the final loss only use self.loss_func (CE), it this right?

**Comment by alvaroqsaldanha** [comment]
hey, I also wanted to know! 

the paper's method is commented out in the official code:

`  
#def get_losses(self, data_dict: dict, pred_dict: dict) -> dict:
`

**Comment by Eurus-W** [comment]
@fire717 @alvaroqsaldanha Hey guys, I'm also working on this. I was wondering what your results were for the two settings (CE only vs. CE + orthogonal + KSV).

For me, adding the orthogonal and KSV losses led to a significant drop in performance. I just wanted to check if you're seeing the same thing.

**Comment by alvaroqsaldanha** [comment]
Hey @Eurus-W, I have not tried training with CE only, but I did find setting the r parameter to 1024-256 to improve performance on my data setup, of course at the cost of training more parameters.

**Comment by skyline-cpu** [comment]
@Eurus-W Hey, I ran into the same issue as well. Did you manage to figure out what’s causing it?

**Comment by UltraCapture** [comment]
> Hey [@Eurus-W](https://github.com/Eurus-W), I have not tried training with CE only, but I did find setting the r parameter to 1024-256 to improve performance on my data setup, of course at the cost of training more parameters.

I also tried to reproduce results on UnivFD(ProGAN training) with r = 1024-256 and CE+Orth+KSV loss, but not to find any performance improvements. I would be grateful if  you could share your training details?

**Comment by Eurus-W** [comment]
> Hey [@Eurus-W](https://github.com/Eurus-W), I have not tried training with CE only, but I did find setting the r parameter to 1024-256 to improve performance on my data setup, of course at the cost of training more parameters.

@alvaroqsaldanha Got it, thanks for your reply!

**Comment by Eurus-W** [comment]
> [@Eurus-W](https://github.com/Eurus-W) Hey, I ran into the same issue as well. Did you manage to figure out what’s causing it?

@yyxb0627 In my experiments, Cross-Entropy (CE) alone performed much better than CE + orthogonal regularization + KSV loss. I'm guessing that the combined loss might require a tricky lambda for the regularization term.

**Comment by xlyyydGH** [comment]
> hi!
> 
> In the effort_detector.py, line142
> 
> ```
> # loss2 = self.compute_weight_loss()
>  # overall_loss = loss + loss2
> ```
> 
> loss2 is not use, so the final loss only use self.loss_func (CE), it this right?

However, the results using loss2 are quite good

**Comment by potato-asuteed-potato** [comment]
Did you train directly on the UFID dataset using the official code? My results on this dataset are not reproducible.

**Comment by potato-asuteed-potato** [comment]
> > Hey [@Eurus-W](https://github.com/Eurus-W), I have not tried training with CE only, but I did find setting the r parameter to 1024-256 to improve performance on my data setup, of course at the cost of training more parameters.
> 
> I also tried to reproduce results on UnivFD(ProGAN training) with r = 1024-256 and CE+Orth+KSV loss, but not to find any performance improvements. I would be grateful if you could share your training details?

Did you train directly on the UFID dataset using the official code? My results on this dataset are not reproducible.
